packages/fx-core/src/plugins/resource/apim/constants.ts

```diff
@@ -431,6 +431,4 @@ export class ApimArmOutput {
   static readonly ServiceResourceId = "apimServiceResourceId";
   static readonly ProductResourceId = "apimProductResourceId";
   static readonly AuthServerResourceId = "apimAuthServiceResourceId";
-  static readonly PublisherEmail = "apimPublisherEmail";
-  static readonly PublisherName = "apimPublisherName";
```

Comment:
Just to want to know what's the motivation to remove these two config values from the provision output?  Because it contains user sensitive info?

Reply:
Sounds good to me. Thanks for the explaination.